clang/test/CXX/drs/dr24xx.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang_cc1 -std=c++98 -triple x86_64-unknown-unknown %s -verify -fexceptions -fcxx-exceptions -pedantic-errors \
```

Comment:
Is triple necessary for the test?

Reply:
I don't think we need `-fexceptions -fcxx-exceptions -Wno-variadic-macros -Wno-c11-extensions` either.